compiler/src/dotty/tools/backend/jvm/GenBCode.scala

```diff
@@ -180,8 +180,7 @@ class GenBCodePipeline(val entryPoints: List[Symbol], val int: DottyBackendInter
             try   { /*withCurrentUnit(item.cunit)*/(visit(item)) }
             catch {
               case ex: Throwable =>
-                ex.printStackTrace()
-                ctx.error(s"Error while emitting ${int.sourceFileFor(item.cunit)}\n${ex.getMessage}")
+                throw new Exception(s"Error while emitting ${int.sourceFileFor(item.cunit)}", ex)
```

Comment:
There's another similar case in https://github.com/lampepfl/dotty/blob/ab42b4d76f50f8efe90d1388b98ae2644b31903b/compiler/src/dotty/tools/backend/jvm/GenBCode.scala#L415

Reply:
Modified